src/config/konnectors.json

```diff
@@ -2171,6 +2171,32 @@
     },
     "source": "git://github.com/konnectors/cozy-konnector-sosh.git#latest"
   },
+  {
+    "slug": "theoldreader",
+    "name": "The Old Reader",
+    "icon": "theoldreader.png",
+    "editor": "Jérémie Bertrand",
+    "vendorLink": "https://theoldreader.com/",
+    "categories": [""],
```

Comment:
Is it normal that it doesn't have any category ?

Reply:
No, i put it in "other" categories, Are you agree with that ?